run.sh

```diff
@@ -0,0 +1,52 @@
+
+#!/bin/sh
+#
+# Run Ylitse app on a device.
+
+#set the API_URL variable
+if [[ ${YLITSE_API_URL} && ${YLITSE_LOGIN_URL} && ${YLITSE_TERMS_URL} && ${YLITSE_USERGUIDE_URL} && ${YLITSE_FEEDBACK_URL} ]]; then
+    API_URL=${YLITSE_API_URL}
+    LOGIN_URL=${YLITSE_LOGIN_URL}
```

Comment:
Would WEB_UI_LOGIN_URL be better? That one is used by mentors to edit their mentor profiles on web ui. This one will be removed someday from the app, when mentors can editt their profiles in the app.

Reply:
Probably a thing that can be done on other PR. login_url has been used before to name that and it makes sense to replicate that naming here.